package.json

```diff
@@ -58,6 +58,7 @@
     "lint-staged": "^8.2.1",
     "markdown-it-codesandbox-embed": "^0.1.0",
     "mermaid": "^8.2.3",
+    "patch-package": "^6.2.2",
```

Comment:
I have not used this in the end - but I think it's a tremendously useful package and it's worth keeping it around (we've been using it in the past temporarily and I expect that we might use it again - IMHO better to keep it than doing install-uninstall dance every time that we actually need it).